packages/inventory-compliance/src/Utilities/hooks/TableTools/TableToolTable.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import { Pagination, PaginationVariant } from '@patternfly/react-core';
+import { Table, TableBody, TableHeader, sortable, fitContent } from '@patternfly/react-table';
+import { TableToolbar, PrimaryToolbar } from '@redhat-cloud-services/frontend-components';
+import useTableTools from '../../Utilities/hooks/TableTools/useTableTools';
+
+
+const severityLevels = [ 'high', 'medium', 'low', 'unknown' ];
+const exampleItems = (count) => {
+    new Array(count).map((count) => {
+        const randomNumber = Math.floor(Math.random() * severityLevels.length);
+        return {
+            severity: severityLevels[ randomNumber ],
+            name: `Test Item ${ count }`
+        };
+    });
+};
+
+/* eslint-disable react/display-name */
+export const exampleColumns = [
+    {
+        title: 'Name',
+        prop: 'name',
+        transforms: [ sortable ],
+        sortByProperty: 'name',
+        renderFunc: (name) => (
```

Comment:
This is inspired by the Inventory table, but I thought about actually naming it `component` as in most cases where the Cell might have more complex content a component would be better to use and the naming would encourage that, while still allowing to pass a function (component).